schema/data/test_setup/external_database_setup.sql

```diff
@@ -0,0 +1,40 @@
+do $$
+  begin
+    RAISE NOTICE '*** BEGIN EXTERNAL TEST DATABASE SETUP ***';
+  end
+$$;
```

Comment:
Is this needed? maybe use `debug` instead of `notice` to avoid making our tests output too verbose?

Reply:
I was using it when configuring the makefile to set up the foreign database & left it in, but it's not really needed. I'll just remove it.